charts/rancher-k3s-upgrader/0.4.0/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.0-alpha1
```

Comment:
@Oats87 @HarrisonWAffel, the `questions.yml` doesn't have a max constraint.

Reply:
confirmed by @jiaqiluo  this is not a release-blocker and should be ok to be fixed in the next release.